tests/transforms/test_broadcast_expand.py

```diff
@@ -11,39 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Unit tests for the broadcast_expand transform.
+"""
+# pylint: disable=import-outside-toplevel
 
 import pytest
 import numpy as np
 import pennylane as qml
 from pennylane import numpy as pnp
 
 
-class RX_broadcasted(qml.RX):
-    """A version of qml.RX that detects batching."""
-
-    ndim_params = (0,)
-    compute_decomposition = staticmethod(lambda theta, wires=None: qml.RX(theta, wires=wires))
-
-
-class RZ_broadcasted(qml.RZ):
-    """A version of qml.RZ that detects batching."""
-
-    ndim_params = (0,)
-    compute_decomposition = staticmethod(lambda theta, wires=None: qml.RZ(theta, wires=wires))
-
-
-dev = qml.device("default.qubit", wires=2)
-"""Defines the device used for all tests"""
```

Comment:
Moved to within the tests, because the exact device that needs to be created differs, when using `"backprop"`, for example.